addons/mail/static/src/core/store_service.js

```diff
@@ -149,10 +149,8 @@ export const storeService = {
     start(env, services) {
         const res = reactive(new Store(env, services));
         onChange(res, "threads", () => res.updateBusSubscription());
-        res.discuss.activeTab = res.isSmall ? "mailbox" : "all";
-        services.ui.bus.addEventListener("resize", () => {
```

Comment:
Isn't it a better design to keep relying on the API of the service (having a bus) rather than introducing a custom on change?

Reply:
Switched back to using the bus.